<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-SEKDPB4" failures="3" tests="5" name="ChromeTest" time="17.728" errors="0" timestamp="2022-10-23T08:52:20 IST">
  <testcase classname="testPack.VerifyHomePage" name="verifyContactToHelpCenter" time="0.911">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [https://www.messenger.com/hel] but found [https://www.messenger.com/help],
	expected [Messenger Help Centr] but found [Messenger Help Centre]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://www.messenger.com/hel] but found [https://www.messenger.com/help],
expected [Messenger Help Centr] but found [Messenger Help Centre]
at testPack.VerifyHomePage.verifyContactToHelpCenter(VerifyHomePage.java:100)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- verifyContactToHelpCenter -->
  <testcase classname="testPack.VerifyRoomsPage" name="verifyContactToHelpCenter" time="1.01"/>
  <testcase classname="testPack.VerifyRoomsPage" name="verifyReturnToMessangerButton" time="2.865">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [https://www.messenger.com] but found [https://www.messenger.com/],
	expected [Messenge] but found [Messenger]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://www.messenger.com] but found [https://www.messenger.com/],
expected [Messenge] but found [Messenger]
at testPack.VerifyRoomsPage.verifyReturnToMessangerButton(VerifyRoomsPage.java:85)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- verifyReturnToMessangerButton -->
  <testcase classname="testPack.VerifyHomePage" name="verifyReturnToMessangerButton" time="2.828">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [https://www.messenger.com] but found [https://www.messenger.com/],
	expected [Messenge] but found [Messenger]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://www.messenger.com] but found [https://www.messenger.com/],
expected [Messenge] but found [Messenger]
at testPack.VerifyHomePage.verifyReturnToMessangerButton(VerifyHomePage.java:83)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- verifyReturnToMessangerButton -->
  <testcase classname="testPack.VerifyHomePage" name="verifyContactToHelpCenter12345" time="0.0">
    <skipped/>
  </testcase> <!-- verifyContactToHelpCenter12345 -->
</testsuite> <!-- ChromeTest -->
